### PR TITLE
Updated benchmark files. Table & Plot left

### PR DESCRIPTION
**Pull request by PyDataBlog**
Almost set for public ANN. The plot and table should be updated with the latest figures.

### CODE REVIEW

File: docs/src/index.md [hunk 1]
```diff
@@ -126,7 +127,7 @@ r.converged             # whether the procedure converged
 - [Hamerly()](https://www.researchgate.net/publication/220906984_Making_k-means_Even_Faster) - Hamerly is good for moderate number of clusters (< 50?) and moderate dimensions (<100?).
 - [Elkan()](https://www.aaai.org/Papers/ICML/2003/ICML03-022.pdf) - Recommended for high dimensional data.
 - [Yinyang()](https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/ding15.pdf) - Recommended for large dimensions and/or large number of clusters.
-- [Coreset()](http://proceedings.mlr.press/v51/lucic16-supp.pdf) - Recommended for very fast clustering of very large datasets, when extreme accuracy is not important.
+- [Coreset()](http://proceedings.mlr.press/v51/lucic16-supp.pdf) - Recommended for very fast clustering of very large datasets, when extreme accuracy is not important. *Experimental Implementation*
```
Comment (Arkoniak) [inline docs/src/index.md]: Why is it experimental? As far as I remember it was complete, tested and working.

**Review by Arkoniak — DISMISSED** [review]
I guess *experimental* can be removed from `Coreset`

File: docs/src/index.md (hunk 1 above)
Comment (PyDataBlog) [reply docs/src/index.md]: What about the slow convergence mechanism? That needs to be fixed?

Comment (Arkoniak) [reply docs/src/index.md]: Slow convergence? I guess I do not remember most of the package :-) As far as I remember, Coresets were the fastest algorithm possible. They have less accuracy, that's true, but this "less" in a sense, that it's hard to achieve ~10^-7 relative tolerance, which is way more accurate than needed in real-life applications. For reasonable tolerance, it is extremely fast.

Comment (PyDataBlog) [reply docs/src/index.md]: That comment was for mini-batch. fixed now :)

**Pull request merged**